AsyncDisplayKit/ASButtonNode.mm

```diff
@@ -491,10 +491,17 @@ - (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
     spec = [ASInsetLayoutSpec insetLayoutSpecWithInsets:contentEdgeInsets child:spec];
   }
   
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
   if (CGSizeEqualToSize(self.preferredFrameSize, CGSizeZero) == NO) {
-    stack.sizeRange = ASRelativeSizeRangeMakeWithExactCGSize(self.preferredFrameSize);
+#if DEBUG
```

Comment:
Yes definitely it should replace PFS. As I said already above I would love to remove support for PFS all internally in ASDK and only have a  deprecated property around. @garrettmoon had a point that it may will make converting code from 1.x to 2.0 a bit harder as you have to convert all at once.

Deprecation usually does not mean you totally remove the functionality from before completely, you just signal that you wanna get rid of the functionality at some point in the future, so for better 1.x to 2 conversion I decided to let the basic functionality in there but deprecate the property. As said I would love to get rid of all functionality / appearances of PFS as soon as possible ...

@levi @Adlai-Holler @garrettmoon  Let's make a decision around it. Should I get rid of all functionality around PFS now and only the deprecated property will be around or let the functionality still in there like now just deprecate the property and print out some debug message about the deprecation.